012-quarkus-kafka-streams/src/test/java/io/quarkus/qe/CommonsTest.java

```diff
@@ -12,12 +12,7 @@
 
 import org.junit.jupiter.api.Test;
 
-import io.quarkus.test.common.QuarkusTestResource;
-import io.quarkus.test.junit.QuarkusTest;
-
-@QuarkusTest
-@QuarkusTestResource(StrimziKafkaResource.class)
-public class AlertMonitorTest {
+public abstract class CommonsTest {
```

Comment:
CommonsTest does not say much about what the tests are about. From my point of view, a better name could be `BaseAlertMonitorTest`.